UPGRADE-14.0.md

```diff
@@ -131,6 +131,10 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   method `\Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\FriendlyUrlFacade::removeFriendlyUrlsForAllDomains()` has been removed without a replacement
 -   see #project-base-diff to update your project
 
+#### fixed wrong columns used in migration Version20240102112523 ([#3402](https://github.com/shopsys/shopsys/pull/3402))
```

Comment:
personally, I would rather amend the existing upgrade notes for https://github.com/shopsys/shopsys/pull/2952. I would add there the warning and a note "see also [#2952](2952) where the migration was fixed". However, I do not insist on that